Library/.rubocop.yml

```diff
@@ -295,6 +295,14 @@ Sorbet/StrictSigil:
   Include:
     - "**/*.rbi"
 
+Sorbet/TrueSigil:
+  Enabled: true
+  Exclude:
+    - "Taps/**/*"
+    - "/**/{Formula,Casks}/**/*.rb"
+    - "**/{Formula,Casks}/**/*.rb"
+    - "Homebrew/test/**/*.rb"
```

Comment:
But we just removed [all of the rest of the non-taps exclusions](https://github.com/Homebrew/brew/issues/14685). 😅

Reply:
Not sure that I'm following – this mirrors the `Sorbet/FalseSigil` config above it

Reply:
Yes, I was being facetious. Two of these is fine.